hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/merkle/MerkleTopic.java

```diff
@@ -304,6 +304,10 @@ public void setMemo(final @Nullable String memo) {
         this.memo = ((null != memo) && !memo.isEmpty()) ? memo : null;
     }
 
+    public int getNumber() {
```

Comment:
Instead of exposing `getNumber` pull it out of the `EntityNum` via `getKey`.  (Better to not have to modify this class.)

Reply:
Fixed by adding [topic.getKey().intValue()](https://github.com/hashgraph/hedera-services/blob/6ff5a597ead7233a987408ecc4f13fa4d09b3b47/hedera-node/cli-clients/src/main/java/com/hedera/services/cli/signedstate/DumpTopicsSubcommand.java#L116). Removed getNumber from `MerkleTopic`